data/model/user.py

```diff
@@ -324,6 +324,10 @@ def update_enabled(user, set_enabled):
 
 def create_robot(robot_shortname, parent, description="", unstructured_metadata=None, token=None):
     (username_valid, username_issue) = validate_username(robot_shortname)
+    if config.app_config.get("ROBOTS_DISALLOW", False):
+        msg = "Robot account has been disabled. Please contact your administrator."
```

Comment:
Should it be `Robot accounts have been disabled`?

Reply:
agreed ... but let's first get the Team triage if the change is something we want to consider as it disabled mirroring of repositories as well.